guidelines/templates/resources/getting-started.md

```diff
@@ -0,0 +1,29 @@
+---
+title: {Document title}
+type: Getting Started
+---
+
+>**NOTE:** Blockquotes in this document provide instructions. Remove them from the final document. Add more sections to the document if you need to explain a given aspect in more details. Use H2 (##) to introduce a new section.
+>
+>This is a template for the **Getting Started** document type that provides a clear step-by-step instruction that helps the user to understand a given concept better. The user must be able to go through all the steps of the document and complete them.
+
+>  In the document:
+> * Start the document with the introductory sentence presented in this template.
+> * Follow the `{000}-gs-{document-title}.md` convention to name the document. The title must summarize what it is about.
+>
+> For reference, see the existing **Getting Started** document concerning the [Sample service deployment on the Kyma cluster](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/034-gs-sample-service-deployment-to-cluster.md).
+
+
+This Getting Started guide shows {describe the purpose of the Getting Started guide}.
+
+## Prerequisites
+
+> List the requirements to run the chart.
+
+## Steps
+
+> Provide the steps that the user must follow to go through the Getting Started guide. Use an ordered list to introduce the steps. Add more sub-sections if you need to explain a given aspect in more details. Use H3 (###) to introduce a new sub-section.
```

Comment:
follow to go through the Getting Started guide -> I guess it is rather about completing a certain action/receiving a specific result rather than completing the guide itself.

Reply:
Changed